src/cc3200.rs

```diff
@@ -72,11 +72,15 @@ impl Board {
     }
 
     pub fn disable_irq() {
-        unsafe { asm!("cpsid i"); }
+        unsafe {
+            asm!("cpsid i");
+        }
```

Comment:
This comment was just triggered by me seeing the disable_irq prototype and not really relevant to this PR directly.

I actually dislike the raw enable/disable_irq functions since it's almost always a mistake to call them directly.

This is even more so on the Cortex M family where interrupts have priorities and you can disable a priority and all lower priorities.

So I'd rather see disable_irq return the current PRIMASK setting, and have enable_irq take that PRIMASK as an argument. In C, you would do something like this:
```
irq_state_t  state = disable_irq();
... do some stuff ...
enable_irq(state);
```